fw/cache.c

```diff
@@ -3257,9 +3296,10 @@ tfw_cfgop_cache_methods(TfwCfgSpec *cs, TfwCfgEntry *ce)
 			return -EINVAL;
 		}
 		if (__cache_method_nc_test(method)) {
-			T_WARN_NL("%s: non-cacheable method '%s' is set "
+			T_ERR_NL("%s: non-cacheable method '%s' is set "
 				  "as cacheable\n",
 				  cs->name, val);
+			return -EINVAL;
```

Comment:
It totally breaks our [cache_methods](https://github.com/tempesta-tech/tempesta/wiki/Caching-Responses#cache_methods). We must discuss it.

Reply:
After our discussion we decide to do it in this way